_data/projects/Hoodie.yml

```diff
@@ -0,0 +1,11 @@
+name: Hoodie
+desc: Generic Backend for frontend Apps
+site: https://github.com/dahlbyk/up-for-grabs.net
+tags:
+- javascript
+- web
+- offlinefirst
+- nobackend
+upforgrabs:
+  name: starter
+  link: http://go.hood.ie/hoodie-starter-issues
```

Comment:
We've got discussions about tracking multiple repositories #244 and multiple labels #240 but this is a fine starting point